### DISCUSSION TITLE
How to create a overlapping image

### DISCUSSION THREAD

**Discussion by ssgwshota** [satijalab/seurat#5024] [Q&A]
Hi
FeaturePlots have the Blend function, but SpatialFeaturePlots do not.
For example, how can I superimpose the two genes of 
SpatialFeaturePlot(brain, features = c("Penk","Pde10a"))
 with different colors?

**Comment by Plundgren1** [comment]
I have the same question!

**Reply by qqspm** [reply]
Need the function as well!
